src/include/txn.h

```diff
@@ -153,13 +181,13 @@ struct __wt_txn {
 
 	/* Checkpoint status. */
 	WT_LSN		ckpt_lsn;
-	bool		full_ckpt;
 	uint32_t	ckpt_nsnapshot;
 	WT_ITEM		*ckpt_snapshot;
+	bool		full_ckpt;
```

Comment:
This is irrelevant to the rest of the change: I started reordering fields to save space.  If we moved `ckpt_nsnapshot` after `ckpt_snapshot`, I think it would save 8 bytes per session...